package.json

```diff
@@ -45,8 +127,9 @@
     "test": "./run",
     "lint": "eslint . run",
     "types": "tsc",
-    "clean": "rimraf *.d.ts doc",
-    "prepublishOnly": "npm run clean && npm run types"
+    "build": "babel --out-dir dist/ --out-file-extension .mjs assert_utils.js browser_console.js browser_utils.js config.js curl_command.js email.js external_locking.js index.js loader.js locking.js main.js net_utils.js output.js promise_utils.js render.js results.js runner.js utils.js version.js",
```

Comment:
Maintaining this file list and the `exports` block above will be pretty important - maybe that should be mentioned in a CONTRIBUTING.md? Could be a start at least.

That should also include a note about using require in this repo, e.g. to never use inline sync require (as mentioned in the comment in version.js)

Reply:
I'm hoping to remove the file mess very soon by moving all files into an `src/` directory. Didn't want to do it with this PR as the diff would become too hard to follow.

I'll add a note about inline `require` calls 👍